Drafts/scs-XXXX-v1-harbor-deployment.md

```diff
@@ -0,0 +1,141 @@
+decision record [Requirements for container registry](https://github.com/SovereignCloudStack/standards/blob/main/Standards/scs-0212-v1-requirements-for-container-registry.md).
+It specified requirements for the container registry implementation that need to be
+fulfilled in the context of SCS and also made an architectural decision on what
+implementation should be used by the SCS for e.g. reference installation. As a result,
+Harbor project has been selected as an appropriate container registry implementation for
+the SCS ecosystem.
+
+## Motivation
+
+This proposal is motivated by use cases in which cloud service providers (CSP) would
+like to offer private container registries to their customers. The specific use cases
+should be discussed (see [Open questions](#open-questions)), but our assumption is that
+there are use cases, where CSP offers a recipe (maintained by SCS) for customers to
+deploy the private registry themselves utilizing CSP infrastructure or CSP offers
+container registry as a service. In both cases, the target platform is the Kubernetes
+cluster that operates on top of CSP IaaS.
+
+The purpose of this document is to investigate Harbor deployment possibilities (for the
+Kubernetes cluster as a target platform), compare their supported features, and then
+make an architectural decision on what Harbor deployment strategy will be recommended by the SCS.
+
+## Design Considerations
+
+### Options considered
+
+#### Harbor Helm Chart
+
+[Harbor Helm][harbor-helm] project contains a helm chart for Harbor deployment to the
+target Kubernetes cluster. Harbor helm chart package contains collections of template
+files that describe the installation of Harbor fundamental services and related components
+(e.g. databases) (see [Architecture-Overview-of-Harbor](https://github.com/goharbor/harbor/wiki/Architecture-Overview-of-Harbor) for further details).
+
+The repository is located under the [goharbor](https://github.com/goharbor) GitHub
+organization and the official Harbor documentation mentions it as a way to deploy
+[Harbor on Kubernetes](https://goharbor.io/docs/edge/install-config/harbor-ha-helm/).
+The project is driven and maintained by the Harbor community, and it is
+up-to-date with the latest Harbor releases. Its ability to deploy and manage Harbor
+components are listed below:
+
+- Deploy Harbor fundamental services (e.g. portal, core, jobservice, etc.)
+- Deploy Harbor fundamental services in HA mode
+- Deploy Harbor data access layer (PostgreSQL, Redis). Object store
+  deployment is not supported.
+- Deploy Harbor services exposed with a load balancer
+- Ability to upgrade the Harbor registry version
+
+#### Harbor Operator
+
+[Harbor Operator][harbor-operator] project provides a solution to deploy and manage a
+full Harbor service stack including both Harbor fundamental components and their
+relevant dependent services such as database, cache, and storage services to the target Kubernetes
```

Comment:
Not sure, but I think this current wording may be unclear on what depends on what.

Suggesting:

```suggestion
relevant dependencies such as database, cache, and storage services to the target Kubernetes
```